src/Service/MigrationService.php

```diff
@@ -0,0 +1,638 @@
+     * Default is false, in other words the output is fully verbose by default.
+     */
+    public function compact(bool $value): void
+    {
+        $this->compact = $value;
+    }
+
+    /**
+     * Column definition strings used for creating migration code.
+     *
+     * The format of each definition is `COLUMN_NAME:COLUMN_TYPE:COLUMN_DECORATOR`. Delimiter is `,`. For example,
+     * `--fields="name:string(12):notNull:unique"` produces a string column of size 12 which is not null and unique
+     * values.
+     *
+     * Note: primary key is added automatically and is named id by default. If you want to use another name you may
+     * specify it explicitly like `--fields="id_key:primaryKey,name:string(12):notNull:unique"`
+     *
+     * @param array $value
+     */
+    public function fields(array $value): void
+    {
+        $this->fields = $value;
+    }
+
+    /**
+     * Set maximum length of a migration name.
+     *
+     * @param int $value
+     */
+    public function maxNameLength(int $value): void
+    {
+        $this->maxNameLength = $value;
+    }
+
+    /**
+     * List of namespaces containing the migration classes.
+     *
+     * @param array $value
+     *
+     * Migration namespaces should be resolvable as a [path alias](guide:concept-aliases) if prefixed with `@`, e.g.
+     * if you specify the namespace `app\migrations`, the code `$this->aliases->get('@app/migrations')` should be able
+     * to return the file path to the directory this namespace refers to.
+     *
+     * This corresponds with the [autoloading conventions](guide:concept-autoloading) of Yii.
+     */
+    public function migrationNamespaces(array $value): void
+    {
+        $this->migrationNamespaces = $value;
+    }
+
+    /**
+     * The directory containing the migration classes.
+     *
+     * This can be either a [path alias](guide:concept-aliases) or a directory path.
+     *
+     * Migration classes located at this path should be declared without a namespace.
+     * Use {@see migrationNamespaces} property in case you are using namespaced migrations.
+     *
+     * If you have set up {migrationNamespaces}, you may set this field to `null` in order to disable usage of
+     * migrations that are not namespaced.
+     *
+     * In general, to load migrations from different locations, {migrationNamespaces} is the preferable solution as the
+     * migration name contains the origin of the migration in the history, which is not the case when using multiple
+     * migration paths.
+     *
+     * @param array $value
+     *
+     * {@see $migrationNamespaces}
+     */
+    public function migrationPath(array $value): void
+    {
+        $this->migrationPath = $value;
+    }
+
+    /**
+     * Set the name of the table for keeping applied migration information.
+     *
+     * @param string $value the name of the table for keeping applied migration information.
+     */
+    public function migrationTable(string $value): void
```

Comment:
Not used?

Reply:
can be used to change the default migration table.